README.md

```diff
@@ -1,3 +1,5 @@
+[![Coverage Status](https://coveralls.io/repos/github/andela/Ah-backend-aquaman/badge.svg?branch=master)](https://coveralls.io/github/andela/Ah-backend-aquaman?branch=master)
```

Comment:
Thanks, @BagzieGracious ,I think you want to add the branch as develop as that's the one we will be merging to.

Reply:
Yeah, that is True